docs/components/connectors/custom-built-connectors/connector-sdk.md

```diff
@@ -843,31 +843,52 @@ and how to invoke them. This component is the runtime environment specific to Ca
 
 Regarding Self-Managed environments, you are responsible for providing the runtime environment that
 can invoke the Connectors. The Connector SDK provides a
-[pre-packaged environment](#pre-packaged-runtime-environment) and a
-[Connector job handler](#connector-job-handler) to make this situation as convenient as possible.
+[pre-packaged environment](#pre-packaged-runtime-environment) and means to create a
+[custom environment](#custom-runtime-environment) to make this situation as convenient as possible.
 
 ### Pre-packaged runtime environment
 
 The SDK comes with a pre-packaged runtime environment that allows you to run select Connector runtimes
 as local job workers out-of-the-box. You can find this Java application on
-[Maven Central](https://search.maven.org/artifact/io.camunda.connector/connector-runtime-job-worker).
+[Maven Central](https://search.maven.org/artifact/io.camunda/spring-zeebe-connector-runtime).
 
 Refer to the [Self-Managed installation guide](/self-managed/connectors-deployment/install-and-start.md) for details on how to
 set up this runtime environment.
 
-### Connector job handler
+### Custom runtime environment
 
 If using the pre-packaged runtime environment that comes with the SDK does not fit your use case,
-you can create a custom runtime environment. To wrap [Connector functions](#runtime-logic) as job
-workers conveniently, the SDK provides the wrapper class `ConnectorJobHandler`.
+you can create a custom runtime environment. There are three options that come with the SDK:
+
+- Create a custom job worker using Spring Zeebe.
+- Wrap Connector functions as job workers using the `ConnectorJobHandler`.
+- Implement your own Connector function wrapper.
+
+#### Spring Zeebe
+
+Being a Spring-wrapper around the [Zeebe Java client](/apis-clients/java-client/index.md), Spring Zeebe supports
+[running outbound Connectors](https://github.com/camunda-community-hub/spring-zeebe/#run-outboundconnectors) out of the box.
+You can expose them as Spring beans in your Spring application and Spring Zeebe picks them up and runs them automatically.
+Using this approach, you can run multiple Connector functions in one job worker application.
```

Comment:
Yeah makes sense. 👍 I had it that way first actually but wasn't sure about it. I used job worker application then but since you stumbled over this now as well, I'll go for Java application 🙂

Reply:
```suggestion
Using this approach, you can run multiple Connector functions in one Java application.
```